LiteCore/Query/QueryParser.cc

```diff
@@ -2042,7 +2049,7 @@ namespace {
     }
 
     // Returns the FTS table name given the LHS of a MATCH expression.
-    string QueryParser::FTSTableName(const Value *key) const {
+    pair<string, string> QueryParser::FTSTableName(const Value *key) const {
```

Comment:
The comment description about needs to update as the return type is changed?

Reply:
done